### PR TITLE
Addition multiple target platforms

### PR DESCRIPTION
**Pull request by alspozdnyakov**

### CODE REVIEW

File: Behavioral.Automation.DemoScenarios/Behavioral.Automation.DemoScenarios.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```
Comment (Brrovko) [inline Behavioral.Automation.DemoScenarios/Behavioral.Automation.DemoScenarios.csproj:4]: What about Behavioral.Automation.csproj ?

**Review by alexhurs — APPROVED** [review]

**Review by ILastovka — APPROVED** [review]

File: Behavioral.Automation.DemoScenarios/Behavioral.Automation.DemoScenarios.csproj (hunk 1 above)
Comment (alspozdnyakov) [reply Behavioral.Automation.DemoScenarios/Behavioral.Automation.DemoScenarios.csproj:4]: Updated

File: Behavioral.Automation.DemoBindings/Behavioral.Automation.DemoBindings.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```
Comment (Brrovko) [inline Behavioral.Automation.DemoBindings/Behavioral.Automation.DemoBindings.csproj:4]: Not required for this project.

File: Behavioral.Automation.DemoScenarios/Behavioral.Automation.DemoScenarios.csproj (hunk 1 above)
Comment (Brrovko) [reply Behavioral.Automation.DemoScenarios/Behavioral.Automation.DemoScenarios.csproj:4]: Not required for this project.

File: src/Behavioral.Automation.UnitTests/Behavioral.Automation.UnitTests.csproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```
Comment (Brrovko) [inline src/Behavioral.Automation.UnitTests/Behavioral.Automation.UnitTests.csproj:4]: Not required for this project.

**Review by ILastovka — APPROVED** [review]

File: Behavioral.Automation.DemoBindings/Behavioral.Automation.DemoBindings.csproj [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```
Comment (Brrovko) [inline Behavioral.Automation.DemoBindings/Behavioral.Automation.DemoBindings.csproj:4]: TargetFramework

File: Behavioral.Automation.DemoScenarios/Behavioral.Automation.DemoScenarios.csproj [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```
Comment (Brrovko) [inline Behavioral.Automation.DemoScenarios/Behavioral.Automation.DemoScenarios.csproj:4]: TargetFramework

File: src/Behavioral.Automation.UnitTests/Behavioral.Automation.UnitTests.csproj [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```
Comment (Brrovko) [inline src/Behavioral.Automation.UnitTests/Behavioral.Automation.UnitTests.csproj:4]: TargetFramework

**Review by Brrovko — APPROVED** [review]

**Review by SergeyRuzin — APPROVED** [review]

**Pull request merged**